site/e2e/tests/manageUsers.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from "@playwright/test";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("manage users", async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/users`, { waitUntil: "domcontentloaded" });
+  await expect(page).toHaveTitle("Users - Coder");
+
+  await page.getByRole("button", { name: "Create user" }).click();
+  await expect(page).toHaveTitle("Create User - Coder");
+
+  const userValues = {
+    username: "testuser",
+    email: "testuser@coder.com",
+    loginType: "password",
+    password: "s3cure&password!",
+  };
+  await page.getByLabel("Username").fill(userValues.username);
+  await page.getByLabel("Email").fill(userValues.email);
+  await page.getByLabel("Login Type").click();
+  await page.getByRole("option", { name: "Password", exact: false }).click();
+  // Using input[name=password] due to the select element utilizing 'password'
+  // as the label for the currently active option.
+  const passwordField = page.locator("input[name=password]");
+  await passwordField.fill(userValues.password);
+  await page.getByRole("button", { name: "Create user" }).click();
```

Comment:
Can we create a function similar to `createWorkspace(...)` to reuse it in the next tests? I expect to use it in OpenID Connect and Github tests.

Reply:
I would only abstract it into a function when we need it. I was not planning to test different authentications since it may require some BE changes like having a "dummy" OAuth server.

Reply:
We have already implemented a mocked Github API, right? I think I saw it [here](https://github.com/coder/coder/blob/main/site/e2e/tests/externalAuth.spec.ts#L21).

Reply:
Also, we have to do this too: https://github.com/coder/coder/issues/12585